english/better-call-saul/saul.md

```diff
@@ -0,0 +1,5 @@
+You dial Sauls number...
+
+"Hi, its Saul Goodman!"
+
+Clearly you are not good and you think you are being pranked so you [go back to thinking about what you should do](/english/marshmallow.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.